tests/unit/utils/Utils.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __UTILS_H__
+#define __UTILS_H__
+
+/*
```

Comment:
No that is fine, I was just checking on whether it was copied from somewhere.

Ok last question: there is no standard c++ function to do this thing without warning?